core/services/eth/contracts/FluxAggregator_test.go

```diff
@@ -18,6 +18,27 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+func mustEVMBigInt(t *testing.T, val *big.Int) []byte {
+	ret, err := utils.EVMWordBigInt(val)
+	require.NoError(t, err, "evm BigInt serialization")
+	return ret
+}
+
+func makeRoundStateReturnData(roundID uint64, eligible bool, answer, timesOutAt, availableFunds, paymentAmount uint64) string {
+	var data []byte
+	data = append(data, utils.EVMWordUint64(roundID)...)
+	if eligible {
+		data = append(data, utils.EVMWordUint64(1)...)
+	} else {
+		data = append(data, utils.EVMWordUint64(0)...)
+	}
+	data = append(data, utils.EVMWordUint64(answer)...)
+	data = append(data, utils.EVMWordUint64(timesOutAt)...)
+	data = append(data, utils.EVMWordUint64(availableFunds)...)
+	data = append(data, utils.EVMWordUint64(paymentAmount)...)
+	return "0x" + hex.EncodeToString(data)
```

Comment:
I like using go-ethereum's `hexutil`, where you can do encoding like there and don't have to worry about remembering the '0x' prefix.

Reply:
https://godoc.org/github.com/ethereum/go-ethereum/common/hexutil#Encode